frontend/src/components/filters/AttributeFilters.js

```diff
@@ -11,7 +11,12 @@ import {
 
 const { Search } = Input;
 
-const AttributeFilters = ({ onSearchChange = () => {} }) => {
+const AttributeFilters = ({
+  onSearchChange,
+  handleAttributeFilter,
+  handleAttributeClearFilter,
+  isBulkUplodNotRequired = false,
```

Comment:
bulkUploadButton, default false.

Then search for AttributeFilters in src folders. Add bulkUploadButton params to it.